hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,5 @@
+logger.notice("Updating ownership of the evr extension if it is enabled")
+result = execute!("runuser -l postgres -c \"psql -d foreman -c \\\"SELECT 1 from pg_extension WHERE extname='evr';\\\"\"", false, true)
+unless (result.include?('(0 rows)'))
+  execute!("runuser -l postgres -c \"psql -d foreman -c \\\"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\\\"\"", false, true)
+end
```

Comment:
@ekohl : Updated to check if extension evr exists in db. This check seems more relevant to updating ownership. The update ownership query itself is idempotent and doesn't fail if evr extension isn't enabled so we don't need any checks before we run it but we can keep the checks if preferred.

Let me know if this works. I'll make a push to remove the same from the postgresql upgrade file.